test/common.py

```diff
@@ -656,6 +656,9 @@ def require_wasm_eh(self):
       self.fail('either d8 or node >= 16 required to run wasm-eh tests.  Use EMTEST_SKIP_EH to skip')
 
   def require_jspi(self):
+    if not self.is_wasm():
```

Comment:
@sbc100 It looks like we weren't running any JSPI test on wasm2js before. I've added this to skip.

Reply:
sgtm